File: Com.Fubei.Api.OpenApi/Biz/MerchantApi/Model/Response/OrderScanResultEntity.cs
using System;
using System.Collections.Generic;
using System.Text;
using Com.Illuminati.Galileo.Model;
using Newtonsoft.Json;

namespace Com.Fubei.Api.OpenApi.Biz.MerchantApi.Model.Response
{
    public class OrderScanResultEntity : BaseEntity
    {
        [JsonProperty("order_sn")]
        public string OrderSn { get; set; }

        [JsonProperty("trade_no")]
        public string TradeNo { get; set; }

        [JsonProperty("qr_code")]
        public string QrCode { get; set; }

        [JsonProperty("store_id")]
        public int StoreId { get; set; }

        [JsonProperty("cashier_id")]
        public int? CashierId { get; set; }

        [JsonProperty("user_id")]
        public string UserId { get; set; }
    }
}

File: Com.Fubei.OpenApi.Vb/Biz/MerchantApi/Model/Response/OrderScanResultEntity.vb
Imports Com.Illuminati.Galileo.Model
Imports Newtonsoft.Json

Namespace Biz.V1.Model.Response
    Public Class OrderScanResultEntity
        Inherits BaseEntity

        <JsonProperty("order_sn")>
        Public Property OrderSn As String

        <JsonProperty("trade_no")>
        Public Property TradeNo As String

        <JsonProperty("qr_code")>
        Public Property QrCode As String

        <JsonProperty("store_id")>
        Public Property StoreId As Integer

        <JsonProperty("cashier_id")>
        Public Property CashierId As Integer?

        <JsonProperty("user_id")>
        Public Property UserId As String
    End Class
End Namespace
